src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -1021,7 +1022,8 @@ namespace SurfaceGeometry {
             state.dataSurface->SurfHighTempErrCount(SurfNum) = 0;
             state.dataSurface->SurfIntConvCoeffIndex(SurfNum) = ConvectionConstants::HcInt_SetByZone;
             state.dataSurface->SurfExtConvCoeffIndex(SurfNum) = 0;
-            state.dataSurface->SurfTAirRef(SurfNum) = 0;
+            state.dataSurface->SurfTAirRef(SurfNum) = DataSurfaces::RefAirTemp::Invalid;
+            state.dataSurface->SurfTAirRefRpt(SurfNum) = static_cast<int>(DataSurfaces::RefAirTemp::Invalid);
```

Comment:
Is this supposed to use SurfTAirRefReportVals?

Reply:
No. I guess not. There is no value for invalid.

Reply:
Once I set -1 (`static_cast<int>(DataSurfaces::RefAirTemp::Invalid)`) as a initial value.